hail/python/hailtop/batch/backend.py

```diff
@@ -117,8 +118,7 @@ def _run(self,
 
         tmpdir = self._get_scratch_dir()
 
-        lines = ['#!/bin/bash',
-                 'set -e' + 'x' if verbose else '',
+        lines = ['set -e' + 'x' if verbose else '',
```

Comment:
when `shell=True` the following occurs:

`Popen(['/bin/sh', '-c', args[0], args[1], ...])`. The shebang has no effect.

Reply:
Yep